osquery/events/darwin/endpointsecurity_fim.cpp

```diff
@@ -85,26 +87,93 @@ void EndpointSecurityFileEventPublisher::configure() {
     auto events = sc->es_file_event_subscriptions_;
 
     for (const auto& p : muted_path_literals_) {
-      auto result = es_mute_path_literal(es_file_client_, p.c_str());
-      if (result == ES_RETURN_ERROR) {
+      es_return_t rc;
+      if (__builtin_available(macos 13.0, *)) {
+        rc =
+            es_mute_path(es_file_client_, p.c_str(), ES_MUTE_PATH_TYPE_LITERAL);
+      } else {
+        rc = es_mute_path_literal(es_file_client_, p.c_str());
+      }
+      if (rc == ES_RETURN_ERROR) {
         VLOG(1) << "Unable to mute path literal: " << p;
       }
     }
 
     for (const auto& p : muted_path_prefixes_) {
-      auto result = es_mute_path_prefix(es_file_client_, p.c_str());
-      if (result == ES_RETURN_ERROR) {
+      es_return_t rc;
+      if (__builtin_available(macos 13.0, *)) {
+        rc = es_mute_path(es_file_client_, p.c_str(), ES_MUTE_PATH_TYPE_PREFIX);
+      } else {
+        rc = es_mute_path_prefix(es_file_client_, p.c_str());
+      }
+      if (rc == ES_RETURN_ERROR) {
         VLOG(1) << "Unable to mute path with prefix: " << p;
       }
     }
 
     for (const auto& p : default_muted_path_literals_) {
-      auto result = es_mute_path_literal(es_file_client_, p.c_str());
-      if (result == ES_RETURN_ERROR) {
+      es_return_t rc;
+      if (__builtin_available(macos 13.0, *)) {
+        rc =
+            es_mute_path(es_file_client_, p.c_str(), ES_MUTE_PATH_TYPE_LITERAL);
+      } else {
+        rc = es_mute_path_literal(es_file_client_, p.c_str());
+      }
+      if (rc == ES_RETURN_ERROR) {
         VLOG(1) << "Unable to mute default path: " << p;
       }
     }
 
+    if (__builtin_available(macos 13.0, *)) {
+      auto parser = Config::getParser("file_paths");
+      if (parser != nullptr) {
+        const auto& doc = parser->getData();
+        if (doc.doc().HasMember("exclude_paths")) {
```

Comment:
nit: Instead of checking if a member exists and then accessing it again, use `FindMember`, which will find the member and immediately provide a iterator/pointer to it.